google/cloud/aiplatform/prediction/local_endpoint.py

```diff
@@ -136,6 +136,7 @@ def __init__(
             ValueError: If both of gpu_count and gpu_device_ids are set.
         """
         self.container = None
+        self.container_exited = False
```

Comment:
Since we're managing the container state, maybe we want it to be an enum? What states can the container be in? What actions should be possible in each state?

Reply:
We are actually maintaining a running container or none. I think this is enough.